src/Util/Traits/StreamedResponseTrait.php

```diff
@@ -16,18 +16,24 @@
 
 namespace Pimcore\Bundle\StudioBackendBundle\Util\Traits;
 
+use League\Flysystem\FilesystemException;
+use League\Flysystem\FilesystemOperator;
 use Pimcore\Bundle\StudioBackendBundle\Exception\ElementStreamResourceNotFoundException;
 use Pimcore\Bundle\StudioBackendBundle\Util\Constants\HttpResponseHeaders;
 use Pimcore\Model\Asset;
+use Pimcore\Model\Asset\Video;
 use Symfony\Component\HttpFoundation\StreamedResponse;
 
 /**
  * @internal
  */
 trait StreamedResponseTrait
 {
+    /**
+     * @throws ElementStreamResourceNotFoundException
+     */
     protected function getStreamedResponse(
-        Asset $element,
+        Asset|Video\ImageThumbnailInterface $element,
```

Comment:
Can we move the Video part to the use section and only have the ImageThumbnailInterface here?